src/vsr/replica.zig

```diff
@@ -3926,43 +3977,36 @@ pub fn ReplicaType(
             }
 
             // Request any missing or disconnected headers:
-            if (self.commit_min != self.op) {
```

Comment:
Could you adjust the comment on `fn repair` to describe the overall flow. I find myself a bit lost in which order we repair things in. I _think_ what happens is that: 

* we request the latest prepare `if (self.op < commit_max_limit)` to learn the hash of the latest commited op
* we request missing headers in reverse-chronological order
* we reuqest missing prepares in chronological order

Reply:
Adding:

```
        /// Repair. Each step happens in sequence — step n+1 executes when step n is done.
        ///
        /// 1. Advance the head op to `op_repair_max = min(op_checkpoint_trigger, commit_max)`.
        ///    To advance the head op we request+await a SV. Either:
        ///    - the SV's "hook" headers include op_checkpoint_trigger (if we are ≤1 wrap behind), or
        ///    - the SV is too far ahead, so we will fall back from WAL repair to state transfer.
        /// 2. Acquire missing or disconnected headers in reverse chronological order, backwards from
        ///    op_repair_max.
        ///    A header is disconnected if it breaks the chain with its newer neighbor to the right.
        /// 3. Repair missing or corrupt prepares in chronological order.
        ///    If we are the primary, this may result in nack/truncation before we start the view.
        /// 4. Commit up to op_repair_max. If committing triggers a checkpoint, op_repair_max
        ///    increases, so go to step 1 and repeat.

```

Reply:
Ah, just realized that the reference to the "hook" headers refers to code that isn't in this PR :facepalm: 
Ignore that for now.